packages/shared/src/services/__tests__/ClaudeStreamingService.test.ts

```
@@ -0,0 +1,698 @@
+        createSession: (sessionId?: string) =>
+          Effect.gen(function* () {
+            const id = sessionId || `session-${Date.now()}-${Math.random().toString(36).substr(2, 9)}`;
+            const session: StreamingSession = {
+              id,
+              messages: [],
+              isActive: true,
+              lastActivity: Date.now()
+            };
+            sessions.set(id, session);
+            return session;
+          }),
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)